cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java

```diff
@@ -298,6 +300,8 @@ protected void configure() {
     dsOpService.startAndWait();
     datasetService = injector.getInstance(DatasetService.class);
     datasetService.startAndWait();
+    namespaceService = injector.getInstance(NamespaceService.class);
```

Comment:
Guess: I guess TestFrameworkTest breaks without this since there isn't a default namespace. I think I remember talking about this..